feature-detects/css/flexgap.js

```diff
@@ -0,0 +1,32 @@
+/*!
+{
+  "name": "Flex Gap",
+  "property": "flexgap",
+  "caniuse": "flexbox-gap",
+  "tags": ["css", "flexbox"],
+  "notes": [{
+    "name": "W3C Spec",
+    "href": "https://www.w3.org/TR/css-align-3/#gaps"
+  }]
```

Comment:
you can also add yourself as an author here

Reply:
Thanks!